server/integration_tests/test_data/e2e_edge_cases2/countiesincaliforniawhereincomeisover50000/chart_config.json

```diff
@@ -9,43 +9,31 @@
               {
                 "tiles": [
                   {
-                    "barTileSpec": {
-                      "maxPlaces": 15,
-                      "maxVariables": 15,
-                      "sort": "DESCENDING"
-                    },
-                    "comparisonPlaces": [
-                      "geoId/06085",
-                      "geoId/06081",
-                      "geoId/06041",
-                      "geoId/06075",
-                      "geoId/06001",
-                      "geoId/06013",
-                      "geoId/06061"
-                    ],
+                    "description": "Non-Institutionalized Civilian Adults With No Health Insurance, Earnings 4,999 USD or Less in Los Angeles County",
```

Comment:
The query string is "income 50000", with the new stat var description, we have $50000 and $5000 exist in the description now.

if you change the query to "Counties in California where income is over 60000", it will be different. So i think the issue here is to handle the numbers in the query?